pkgs/development/python-modules/clingo/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3,
+  python3Packages,
+  clingo,
+}:
+
+python3Packages.buildPythonPackage rec {
+  inherit (clingo) pname version src;
+
+  postPatch = ''
+    cd libpyclingo
+    python compile.py h
+  '';
+
+  pyproject = true;
+  build-system = [ python3Packages.setuptools ];
+
+  dependencies = with python3Packages; [ cffi ];
```

And then you can unqualify these

```suggestion
  build-system = [ setuptools ];

  dependencies = [ cffi ];
```